forge-gui/res/cardsfolder/upcoming/saheeli_filigree_master.txt

```diff
@@ -0,0 +1,16 @@
+Name:Saheeli, Filigree Master
+ManaCost:2 U R
+Types:Legendary Planeswalker Saheeli
+Loyalty:3
+A:AB$ Scry | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | ScryNum$ 1 | SubAbility$ DBDraw | SpellDescription$ Scry 1. You may tap an untapped artifact you control. If you do, draw a card.
+SVar:DBDraw:DB$ Draw | Cost$ tapXType<1/Artifact>
```

Comment:
DB doesn't support Cost$
You can try AB$ otherwise UnlessCost should get the job done

Reply:
It crashed with AB so gonna try with unlesscost